x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/process/AnalyticsProcessManager.java

```diff
@@ -168,26 +161,31 @@ private void processData(DataFrameAnalyticsTask task, DataFrameAnalyticsConfig c
             refreshDest(config);
             LOGGER.info("[{}] Result processor has completed", config.getId());
         } catch (Exception e) {
-            if (task.isStopping() == false) {
-                String errorMsg = new ParameterizedMessage("[{}] Error while processing data [{}]", config.getId(), e.getMessage())
-                    .getFormattedMessage();
+            String errorMsg =
+                new ParameterizedMessage("[{}] Error while processing data [{}]", config.getId(), e.getMessage()).getFormattedMessage();
+            if (task.isStopping()) {
+                // Errors during task stopping are expected but we still want to log them just in case.
+                LOGGER.debug(errorMsg, e);
```

Comment:
It will also be useful to log we're stopping in this case.

Reply:
Done.